controllers/apply-next/validators.js

```diff
@@ -80,8 +80,25 @@ const dateParts = joi => {
 
 const budgetValidator = joi => {
     return {
-        base: joi.array(),
+        base: joi
```

Comment:
The base can be any joi schema, so the full schema can live in here, rather than needing to be defined when called.